css/modals/device-selection/_device-selection.scss

```diff
@@ -0,0 +1,68 @@
+.device-selection {
+    color: $feedbackInputTextColor;
+    display: flex;
+
+    .device-selectors {
+        margin-bottom: 5px;
+    }
+
+    .device-selection-column {
+        padding: 10px;
+        width: 50%;
+    }
+
+    .device-selection-video-container {
+        background: black;
+        height: 148px;
+        margin-bottom: 5px;
+
+        .video-input-preview {
```

Comment:
Yeah, the mirroring is done with the existing flipVideoX class in use for LocalVideo.js.